github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -1983,6 +1983,178 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: false
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: kubevirt-prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 4h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-podman-in-container-enabled: "true"
+      preset-podman-shared-images: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.28-sig-network
+    skip_branches:
```

I'd suggest to make all presubmits optional at first, so they don't block merges.

```suggestion
    optional: true
    skip_branches:
```

---

I thought when we create jobs with `always_run: false` - they default to `optional: true` ?

---

I just remember that in general the default for `optional` is `false`, but you might be right that in case of `always_run: false` and `run_if_changed` unset it might be `true`.

Let's try and we get to see :smile: